app/controllers/CreateGroupSelectTeamMembersController.scala

```diff
@@ -199,10 +199,10 @@ class CreateGroupSelectTeamMembersController @Inject()
                     )
                   ).toFuture
                 }, (yes: Boolean) => {
-                  sessionCacheService.put(CONFIRM_TEAM_MEMBERS_SELECTED, yes).flatMap { _ =>
                     if (yes)
                       Redirect(controller.showSelectTeamMembers(None, None)).toFuture
                     else {
+                      sessionCacheService.put(CONFIRM_TEAM_MEMBERS_SELECTED, yes).flatMap { _ =>
```

Comment:
not sure why you've moved this. Maybe renaming "yes" would help it make more sense - is this the "do you want to select more team members?" question

Reply:
if you select yes then you would go around again this page would have yes pre-filled to yes which you don't want

Reply:
okay, that makes sense why it's moved.

I was just thinking about the variable name still (as the value wouldn't be "yes" where it is on this line) but it can be updated later

Reply:
ok, good idea as it's not a good choice of variable name. I should have changed it.